### DISCUSSION TITLE
Texture feedback（材质需求）

### DISCUSSION THREAD

**Discussion by Fogy-F** [Fogy-F/Minimalistic-GTNH-repair#2] [Ideas]
If you feel that some textures are not satisfied or need to redraw, you can propose here.
（如果觉得有些材质不满意或者需要重画的可在这里提出）

**Comment by wwuyan** [comment]
希望可以把每一行的电路板保持现在用同种主体色，每一列的电路板逐渐变大（因为一层层套娃合成）。（比较低劣的改图用于示意，请见谅）
![GTNH电路板 - 副本](https://github.com/user-attachments/assets/8f161c5e-1a3b-4149-b6a9-ba3b5dce6a39)

**Reply by Fogy-F** [reply]
感谢你的建议，之前在issues里也有人反馈过。
因为Minimalistic老版本纵向大小采用的是【3214】的等级递增关系。新版本也一直是沿用的。
以下是之前和其他人讨论过的改版，但没有采纳。
<img width="300" alt="Snipaste_2025-04-12_06-33-43" src="https://github.com/user-attachments/assets/4cfd9b12-0896-44ed-8b96-7d2ec3c2e21b" />

**Reply by Fogy-F** [reply]
如果需要的话，之后更新可以在材质包内加一个用于替换修改的文件夹。

**Reply by wwuyan** [reply]
hummm...好像因为电路板本身的纹理不同，这样排列后还是感觉有点乱？
如果搞替代材质的话，要不直接像原版GT那样同列同色，如果可以的话辅以角标显示等级？

**Reply by Fogy-F** [reply]
纹理的话这个没有好的解决方案，每个电路不同的纹理都是原作者遗留下来的，不太好修改。
这是V1.0.4.stable后面的电路板样式，也许你需求的是这种风格？稍微改一下就是你说的同列同色贴合原版了。
<img width="300" alt="Snipaste_2025-04-12_08-39-14" src="https://github.com/user-attachments/assets/e861df76-6a56-4fef-9dbb-af22108ad9da" />

**Reply by Fogy-F** [reply]
角标显示之前有想法，但有点太突兀了还没想好怎么改。
<img width="300" alt="Snipaste_2025-04-12_08-45-29" src="https://github.com/user-attachments/assets/82304d04-ff45-4a0e-971d-4ed6dc0a1939" />

**Reply by wwuyan** [reply]
确实感觉很难改啊（笑哭）

**Reply by wwuyan** [reply]
个人说的同列同色，是指把现在的材质仅更改材质的文件名，让现在同色的电路板在同一列。而这个角标我个人是觉得挺好的没有很突兀

**Reply by Fogy-F** [reply]
如果是这样的话，那任意电路板将同色。之后在试着按照你的想法修改一下。
角标的话测试了一下游戏内显示不是很好。
<img width="600" alt="Snipaste_2025-04-12_09-04-33" src="https://github.com/user-attachments/assets/6415e043-1a10-48e4-b21f-1ba5503eef03" />

**Reply by wwuyan** [reply]
游戏内看角标确实太小了点

**Reply by Fogy-F** [reply]
大概是这种效果？原先的EV和MV就不采用了。
电路板的颜色之后在和原版匹配。
<img width="300" alt="Snipaste_2025-04-12_11-29-50" src="https://github.com/user-attachments/assets/5b617204-1571-49b5-941d-6cef2f4c4069" />

**Reply by ovibomio1** [reply]
其实任意电路板的颜色可以不用改，之前的彩色就挺好的

**Reply by ovibomio1** [reply]
能看一下游戏里的任务书吗

**Reply by ovibomio1** [reply]
凑个热闹嘛，话说如果这样修改的话合成的时候轮换图就不一样了吧

**Reply by Fogy-F** [reply]
服了，这是任务书的效果。
<img width="300" alt="Snipaste_2025-04-12_15-11-09" src="https://github.com/user-attachments/assets/9bd9d0e5-9c8e-4332-91b1-a2fdd8b5d57a" />

轮换图的显示感觉还行吧。你来点建议？
<img width="300" alt="Snipaste_2025-04-12_15-33-01" src="https://github.com/user-attachments/assets/0af16cbb-9fed-4d8e-8602-69e0f28144e8" />

**Reply by ovibomio1** [reply]
也不赖吧，要不你试试把每一行的颜色改成一样的

**Reply by wwuyan** [reply]
> 也不赖吧，要不你试试把每一行的颜色改成一样的

要同色的话不就是现在的样子吗？😂

**Reply by Fogy-F** [reply]
目前是提供四种替换方案。
<img width="600" alt="Snipaste_2025-04-14_07-31-11" src="https://github.com/user-attachments/assets/db4d6574-45ca-414b-ba24-290ba419d96b" />

这是彩角标的截图。原版的颜色讨论了一下还是不太适合，所以只能保持之前的配色。

<img width="300" alt="circuit-A-3" src="https://github.com/user-attachments/assets/1bb8c93c-3012-427f-8a87-068bfd485389" />

**Reply by Fogy-F** [reply]
如果没有建议或者其他问题的话，之后就上传更新。

**Reply by wwuyan** [reply]
wc，太满意了，冲冲冲！

**Reply by Fogy-F** [reply]
好的，repair分支已更新。

**Comment by Suzumi999** [comment]
超级缸的脸可以画成原版那种类似电脑屏幕的样子吗，现在看着像电池一样一下认不出来😭

**Reply by Fogy-F** [reply]
感谢你的反馈，这个材质包的原设计一直都是这样的，目前的话只能提供这种静态的方案。
<img width="300" alt="Snipaste_2025-06-26_19-29-44" src="https://github.com/user-attachments/assets/d09093d6-dae8-4d7e-b8ea-acc31d9a908c" />

不过游戏内也有电脑屏幕的覆盖板。
<img width="300" alt="Snipaste_2025-06-26_19-38-12" src="https://github.com/user-attachments/assets/c03449a0-69b9-4729-b5df-eb1ec145576b" />

**Reply by Suzumi999** [reply]
好的好的，请问有交流群什么的吗。我也想帮忙画一下材质，之前做过UI设计师。

**Reply by Fogy-F** [reply]
目前这个材质包用的人并不是很多所以没有交流群，想帮忙可以随时 git 拉取请求。如果后续想继续交流和提供反馈的话，欢迎联系仓库主页里的本人QQ。

**Reply by Fogy-F** [reply]
那么这种方案可以的话，之后就提交更新。

**Comment by StellaCaerulea** [comment]
GTNH 2.9.0 added CTM for Coils and Plascrete Block and others. We may make some cool CTM too.
https://github.com/GTNewHorizons/GT5-Unofficial/pull/7286 Coil
https://github.com/GTNewHorizons/GT5-Unofficial/pull/7286 Glass
https://github.com/GTNewHorizons/GT5-Unofficial/pull/7019 BlockMachines
